src/test/java/seedu/address/testutil/ElderlyBuilder.java

```diff
@@ -0,0 +1,54 @@
+package seedu.address.testutil;
+
+import seedu.address.model.person.Elderly;
+import seedu.address.model.person.information.Age;
+import seedu.address.model.person.information.Nric;
+import seedu.address.model.person.information.RiskLevel;
+
+/**
+ * A utility class to help with building Elderly objects.
+ */
+public class ElderlyBuilder extends PersonBuilderScaffold<ElderlyBuilder> {
```

Comment:
@heeeyi also implemented her own `Elderly` tests in #51. @heeeyi how do you want to resolve this?

Reply:
I shall wait for @heeeyi 's code to be merged and update mine to match hers

Reply:
Fixed merge conflicts in 132752f80b26b30e24ccc37f337c27359714560a